pkg/volume/testing/testing.go

```diff
@@ -400,8 +400,8 @@ var _ DeviceMountableVolumePlugin = &FakeVolumePlugin{}
 var _ NodeExpandableVolumePlugin = &FakeVolumePlugin{}
 
 func (plugin *FakeVolumePlugin) getFakeVolume(list *[]*FakeVolume) *FakeVolume {
-	volumeList := *list
-	if list != nil && len(volumeList) > 0 {
+	var volumeList []*FakeVolume 
+	if list != nil && volumeList = *list && len(volumeList) > 0 {
```

Comment:
Go does not allow assignments inside of `if`. This does not compile.

Reply:
Thanks for the review. Changes made.